frontend/public/components/pod.jsx

```diff
@@ -193,25 +217,9 @@ const Details = ({obj: pod}) => {
 
     </div>
 
-    <div className="co-m-pane__body">
-      <h1 className="co-section-title">Containers</h1>
-      <div className="row">
-        <div className="co-m-table-grid co-m-table-grid--bordered">
-          <div className="row co-m-table-grid__head">
-            <div className="col-sm-2 col-xs-4">Name</div>
-            <div className="col-sm-2 hidden-xs">Id</div>
-            <div className="col-sm-2 col-xs-8">Image</div>
-            <div className="col-md-2 col-sm-2 hidden-xs">Security Scan</div>
-            <div className="col-md-1 col-sm-2 hidden-xs">State</div>
-            <div className="col-md-1 col-sm-2 hidden-xs">Restart Count</div>
-            <div className="col-md-2 hidden-sm hidden-xs">Started At</div>
-          </div>
-          <div className="co-m-table-grid__body">
-            {pod.spec.containers.map((c, i) => <ContainerRow key={i} pod={pod} container={c} />)}
-          </div>
-        </div>
-      </div>
-    </div>
+    {pod.spec.initContainers ? <ContainerTable heading="Init Containers" containers={pod.spec.initContainers} pod={pod} /> : null}
```

Comment:
I'm good with this since you can set the heading uniquely per table.

Reply:
```jsx
 {pod.spec.initContainers && <ContainerTable heading="Init Containers" containers={pod.spec.initContainers} pod={pod} />}
```

Reply:
+1 this is cleaner.